src/object/srv_obj_migrate.c

```diff
@@ -758,6 +915,22 @@ migrate_update_parity(struct migrate_one *mrone, struct ds_cont_child *ds_cont,
 
 		tmp_sgl.sg_iovs = &tmp_iov;
 		iod->iod_recxs = &tmp_recx;
+
+		rc = csum_init_csummer(csum_iov, &csummer);
+		if (rc != 0) {
+			D_ERROR("Error initializing csummer");
+			D_GOTO(out, rc);
+		}
+
+		rc = alloc_iods_csums(&iod_csums,
+				      iod,
+				      1, csum_iov,
+				      csummer);
+		if (rc != 0) {
+			D_ERROR("Error allocating iods");
+			D_GOTO(out, rc);
+		}
+
```

Comment:
parity buffer is actually recalculated above by obj_ec_encode_buf(), so it needs to re-calculate the csum here, instead of using fetched csum.

Reply:
Yes, this will need to be fixed. but there are failing EC tests (w/ checksums) that make it difficult to verify a change here. I created a new story to resolve this so these changes can land for 1.2. EC + checksums aren't required for 1.2.
DAOS-6811 is the new story.